docs/UserGuide.md

```diff
@@ -392,7 +392,15 @@ This is done as we do not except tutors to teach lessons beyond the day itself.
 </div>
 
 Example:
-* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
+1. You would like to add a new session for **John Doe** on the date **2021-01-01** at **18:00** where you will teach **Biology** for **120** minutes and charge a fee of **$80**.
+2. Type in `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80` to add the session information into TutorBuddy and press <kbd>enter</kbd>.
```

Comment:
Haha for <kbd>enter</kbd>, I think samuel previously mention that we should standardize, so I changed from "enter" over to "Enter" instead. I think you can do that as well ah.

Reply:
Thanks for reminding! Made the changes.